### DISCUSSION TITLE
Achievement question

### DISCUSSION THREAD

**Discussion by eduardconstantin** [drknzz/GitHub-Achievements#21] [Q&A] [answered]
Shouldn't the achievement disappear if the criteria is not met anymore?

I got the starstruck achievement but I don't have any repo that has 16 stars. Is this normal?

**Comment by drknzz** [comment]
That's interesting.
I guess GitHub made it so if you get the achievement it stays forever.
Similiar thing is with Galaxy Brain achievement as when you get it and later you either delete your comments or have them unmarked, the achievement stays.
But in your case it's weird because it seems that just starring a repository and unstarring it right away makes the star count towards the Starstruck achievement.

**Reply by eduardconstantin** [reply]
Oh, so if someone stars and unstars a repo 16 times the achievement is unlocked? This doesn't seems right.
Someone has to speak with GitHub about this.

**Reply by eduardconstantin** [reply]
> so if someone stars and unstars a repo 16 times the achievement is unlocked?

I tested this, doesn't work.

**Comment by 1RB** [comment, marked as answer]
**Once you get an achievement it doesn't go away**

> so if someone stars and unstars a repo 16 times the achievement is unlocked?

no the achievement means that if you have 16 stars already then if that number goes back down then you will still keep your badge

if this helped, please mark this as the answer as I'm trying to get the galaxy brain achievement
